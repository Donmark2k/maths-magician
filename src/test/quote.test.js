import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Quote />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
