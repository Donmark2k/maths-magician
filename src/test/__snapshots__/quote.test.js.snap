// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="quoteMath"
>
  Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. - William Paul, Thurston
</p>
`;
